tensorflow/core/grappler/optimizers/remapper.cc

```diff
@@ -114,6 +116,17 @@ struct FusedBatchNormEx {
   int invalidated = kMissingIndex;
 };
 
+// FusedBatchNormGrad with fused side output and/or activation.
+struct FusedBatchNormGradEx {
+  FusedBatchNormGradEx() = default;
```

Comment:
I don't think this constructor definition does anything.

Reply:
Removed.